context/app/static/js/pages/CellsAPIDemo/DatasetsSelectedByGene.jsx

```diff
@@ -38,29 +39,36 @@ function DatasetsSelectedByGene(props) {
     const { name } = target;
     const setFields = {
       geneName: setGeneName,
-      minGeneExpression: setMinGeneExpression,
-      minCellPercentage: setMinCellPercentage,
     };
     setFields[name](event.target.value);
   }
 
   return (
     <Paper>
```

Comment:
Should we use a form here? There are differing opinions on their necessity.

Reply:
I'm hoping that's something you can sort out when you take these little pieces and make the real thing out of them!